src/Generators/FactoryGenerator.php

```diff
@@ -97,9 +97,10 @@ protected function buildDefinition(Model $model)
                 }
 
                 $class = Str::studly(Str::singular($table));
-                $reference = $this->fullyQualifyModelReference($class) ?? $model;
-
-                $this->addImport($model, $reference->fullyQualifiedNamespace() . '\\' . $class);
+                if (!Str::startsWith($class, '\\')) {
+                    $reference = $this->fullyQualifyModelReference($class) ?? $model;
+                    $this->addImport($model, $reference->fullyQualifiedNamespace() . '\\' . $class);
+                }
```

Comment:
This is wrong.

---

I believe you want to remove the application's namespace prefix for fully-qualified class names that start with '\\'.

eg. for `foreign:\Foo\Bar`

```diff
- use App\Models\\Foo\Bar;
+ use Foo\Bar;
```

To achieve this we need to change:

```diff
$class = Str::studly(Str::singular($table));
$reference = $this->fullyQualifyModelReference($class) ?? $model;

- $this->addImport($model, $reference->fullyQualifiedNamespace() . '\\' . $class);
+ if (Str::startsWith($class, '\\')) {
+     $this->addImport($model, $class);
+ } else {
+     $this->addImport($model, $reference->fullyQualifiedNamespace() . '\\' . $class);
+ }
```
---

This should correctly import **all** fully-qualified class names.